packages/mui-material/src/utils/separators.js

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+/**
+ * It takes a React element and an array of React elements, and returns a new array of React elements
+ * with the separator element inserted between each of the original elements
+ * @param children - The children to add separators to.
+ * @param separator - The separator component to be used.
+ */
+export default function addSeparators(children, separator) {
```

Comment:
I think the word `insert` is more explicit in this case. Also, could you change the file name to `insertSeparator.js` too?
```suggestion
export default function insertSeparator(children, separator) {
```

Reply:
Sounds good, check again please